specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/AppServiceEnvironments.json

```diff
@@ -2163,6 +2253,45 @@
         }
       }
     },
+    "EndpointDependency": {
+      "type": "object",
+      "properties": {
+        "domainName": {
+          "description": "The Domain Name of the dependency.",
+          "type": "string"
+        },
+        "endpointDetails": {
+          "description": "The IP Addresses and Ports used when connecting to DomainName.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EndpointDetail"
+          }
+        }
+      }
+    },
+    "EndpointDetail": {
+      "type": "object",
+      "properties": {
+        "ipAddress": {
+          "description": "An IP Address that Domain Name currently resolves to.",
+          "type": "string"
+        },
+        "port": {
+          "format": "int32",
+          "description": "The port an endpoint is connected to.",
+          "type": "integer"
+        },
+        "latency": {
+          "format": "double",
+          "description": "The time in milliseconds it takes to connect to this IpAddress at this Port.",
+          "type": "number"
+        },
+        "isAccessable": {
+          "description": "Whether it is possible to connect to IpAddress.",
```

Comment:
What does this mean? Whether you are able to make an inbound TCP connection to the ipAddress + port? From where?